spec/spec_helper.rb

```diff
@@ -1,32 +1,18 @@
-if ENV['CI']
-  require 'simplecov'
-  SimpleCov.start
-end
-
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-
-require "manageiq/graphql/schema"
+#
+# General RSpec configuration
+# For Rails or ManageIQ-specific configuration, see manageiq_helper.rb
+# This file is automatically required for all specs
+#
 
 RSpec.configure do |config|
-  config.expect_with :rspec do |expectations|
-    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
-  end
-
-  config.mock_with :rspec do |mocks|
-    mocks.verify_partial_doubles = true
-  end
+end
 
-  config.shared_context_metadata_behavior = :apply_to_host_groups
-  config.filter_run_when_matching :focus
-  config.example_status_persistence_file_path = "spec/examples.txt"
-  config.disable_monkey_patching!
-  config.default_formatter = "doc" if config.files_to_run.one?
-  config.order = :random
+Dir["./spec/support/**/*.rb"].each { |f| require f }
 
-  Kernel.srand config.seed
+require 'manageiq-graphql'
 
-  # arbitrary gems may also be filtered via:
-  # config.filter_gems_from_backtrace("gem name")
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start
```

Comment:
@Fryguy I moved this here because it's generally recommended to start code coverage tools at the very last point you can before the tests begin, for accuracy.

Reply:
Oh wait I remembered that wrong, you're right, it's the total opposite. I'll move it!